### DISCUSSION TITLE
R version

### DISCUSSION THREAD

**Discussion by Spines151** [ianjonsen/foieGras#71] [General]
HI Ian, sorry for another question. I just upgraded to the newest R version, but it seems that aniMotum is not supported in R 4.4.2. In the description, it says, ideally >4.2.0, what is the latest version aniMotum is compatible with? Thanks Christian

**Comment by ianjonsen** [comment]
It works with R 4.4.0, so there should be no reason it won't work with 4.4.2. The Description indicates the minimum (oldest) R version that `aniMotum` has been tested on.

**Reply by Spines151** [reply]
Thanks Ian. It works on 4.4.2 There was an issue on my side.

**Reply by nmasto** [reply]
Similar problems with aniMotum not working and having to revert to older versions. Wonder what your bug was? I keep recieving `Warning in install.packages :
  package ‘aniMotum’ is not available for this version of R

A version of this package for your version of R might be available elsewhere,
see the ideas at
https://cran.r-project.org/doc/manuals/r-patched/R-admin.html#Installing-packages` for R versions 4.4.3

**Reply by ianjonsen** [reply]
`aniMotum` is built & tested on R versions 4.5 (Development), 4.4 and 4.3 on Windows PC, on x86 & arm64 Macs, and on Ubuntu platforms. For each platform, the latest stable OS is used. So as long as you are running R >= 4.1.0 it should install - provided you have the appropriate gfortran and C++ compilers installed, which require RTools (Windows) or Xcode Command Line Tools (Mac) be installed (see Readme). I can't provide further details because I don't know your setup. If you can't build from source, I'd suggest choosing the binary version from R-Universe that best matches your system - [here](https://ianjonsen.r-universe.dev/aniMotum)

Thanks, Ian

**Comment by Spines151** [comment]
Hey 'nmasto', I hope you solved the problem. In my, case it was - as usual - something simple. I tried to install 'animotum' within R - install packages, and I got that error message. Then I used Ian's code on his (animotum) GitHub page, and it installed without problems.
